identity-apps-core/apps/x509-certificate-authentication-portal/src/main/resources/LanguageOptions.properties

```diff
@@ -0,0 +1,30 @@
+# -------------------------------------------------------------------------------------
+#
+# Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
```

Comment:
Let's update the other places as well.

```suggestion
# Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
```

Reply:
https://github.com/wso2/identity-apps/pull/5634/commits/974cdfd8aff3463f185d1cd56298a95febe6f5da